doc/designing-proposal-settings-ui.md

```diff
@@ -209,6 +209,16 @@ incorporated to the main screen doing something like this.
 
 ![screenshot](proposal_settings_ui/idea_brainstorm2.png)
 
+Or this (just two checkboxes)
+
+```
+( ) LVM
+(x) Encryption
```

Comment:
maybe nitpicking but we usually use `[ ]` for checkboxes and `( )` for radio buttons, as ncurses does. It took me a while to get it despite the comment.